README.md

```diff
@@ -55,6 +55,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vite-ts-tailwind-starter](https://github.com/Uninen/vite-ts-tailwind-starter) - Starter template with Vue 3, TypeScript, Tailwind CSS, and Cypress.io e2e tests.
 - [vue-vben-admin-2.0](https://github.com/anncwb/vue-vben-admin) - Starter template with Vue 3, TypeScript, Tailwind CSS, Ant Design Vue2.
 - [electron-vue-next](https://github.com/ci010/electron-vue-next) - Starter template using Vue 3 in Vite and latest Electron with fluent VS Code debug and GitHub release process out-of-box.
+- [vite-components-playground](https://xxholly32.github.io/vite-components-playground/) - A common vue3 components playground using Vite.
```

Comment:
```suggestion
- [vite-components-playground](https://xxholly32.github.io/vite-components-playground/) - A common Vue 3 components playground.
```

And I think it might be better to provide GitHub repo link instead.

Reply:
ok